README.md

```diff
@@ -34,107 +32,138 @@ with header files (and a C++ compiler of course):
 
 Then do:
 
-    gem install taglib-ruby
+```shell
```

Comment:
Not really a fan of the changes in this file. Why does Rubocop care about this file?

Reply:
Actually, some of these changes were due to the markdownlint extension in VSCode.

Reply:
I would undo the `shell` changes in this file, other than that we can merge this PR.

Reply:
Removed `shell` with https://github.com/robinst/taglib-ruby/pull/108/commits/a64d623bd8c87b32f7e63e0287425b43cd874194.